connectivity/lorawan/tests/UNITTESTS/doubles/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-headers-lorawan INTERFACE)
+
+target_include_directories(mbed-headers-lorawan
+    INTERFACE
+        ${mbed-os_SOURCE_DIR}/connectivity/lorawan/include/lorawan
+        ${mbed-os_SOURCE_DIR}/connectivity/lorawan/lorastack
+        ${mbed-os_SOURCE_DIR}/connectivity/lorawan/lorastack/mac
+        ${mbed-os_SOURCE_DIR}/connectivity/lorawan/lorastack/phy
+        ${mbed-os_SOURCE_DIR}/connectivity/lorawan
+        ${mbed-os_SOURCE_DIR}/connectivity/lorawan/system
+)
+
+
+add_library(mbed-stubs-lorawan-headers INTERFACE)
+
+target_include_directories(mbed-stubs-lorawan-headers
+    INTERFACE
+        .
+)
+
+add_library(mbed-stubs-lorawan)
+
+target_compile_definitions(mbed-stubs-lorawan
+    PRIVATE
+        DEVICE_SERIAL=1
+        DEVICE_INTERRUPTIN=1
+        MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE=115200
+        MBED_CONF_LORA_OVER_THE_AIR_ACTIVATION=true
+        MBED_CONF_LORA_AUTOMATIC_UPLINK_MESSAGE=true
+        MBED_CONF_LORA_TX_MAX_SIZE=255
```

Comment:
Can some of the macros be removed from `UNITTESTS/stubs/connectivity/CMakeLists.txt` as we are moving them here?

Reply:
@LDong-Arm Thanks for your review comment, I missed it , will update

Reply:
@LDong-Arm Thanks for your review comment, I will check